core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -93,7 +93,6 @@
 public class TieredBlockStore implements LocalBlockStore
 {
   private static final Logger LOG = LoggerFactory.getLogger(TieredBlockStore.class);
-  private static final long REMOVE_BLOCK_TIMEOUT_MS = 60_000;
```

Comment:
Yeah it provides better information on what the number means... My main concern when removing the constant is to avoid someone bypassing the `getRemoveBlockTimeout` method and uses that constant directly. This could create some problem for testing.

Reply:
I see, is it possible to use WhiteBox to modify the constant or make this one configurable?